Projeto

Giovanirose opened the pull request:
Alterações no texto do projeto!

Giovanirose committed:
Leia.md

Giovanirose committed:
Update README.md

Giovanirose committed:
Add files via upload

Giovanirose committed:
Merge pull request #1 from Giovanirose/primeira-etapa

Primeira etapa

Giovanirose committed:
Merge pull request #2 from Giovanirose/segunda-etapa

Site com parte do PHP implementado

MAmelia commented:
Não consegui abrir os links do mapa de navegação e wireframe. Coloquem os arquivos aqui no git.
Quando um arquivo HTML tem um include, ele precisa ser salvo como PHP e não HTML. Por isso seus includes não funcionam!

Giovanirose commented:
Ah sim! Colocarei-os imediatamente, muito obrigado pela dica do arquivo, espero que não desconte muita nota por essa falta de atencao, irei impletar no codigo agora! Obrigado Professora :)

Giovanirose commented:
Ah entendi agora professora! Vou colocar as imagens assim que possivel, e muito obrigado pela dica! Implementarei imediatamente!

On 11 de out de 2016 5:58 PM -0300, MAmelia notifications@github.com, wrote:

Não consegui abrir os links do mapa de navegação e wireframe. Coloquem os arquivos aqui no git.
Quando um arquivo HTML tem um include, ele precisa ser salvo como PHP e não HTML. Por isso seus includes não funcionam!

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHubhttps://github.com/tec-webII-2h11/giovani-gabrielPagglio-otavio/pull/1#issuecomment-253044048, or mute the threadhttps://github.com/notifications/unsubscribe-auth/ADkXvoDLv4ljgiTtitCivQPoK6a14AaXks5qy_hwgaJpZM4KTh-N.

Giovanirose committed:
Entrega Final - GRAND FINALE

Ola professora boa tarde, enviei todos os arquivos que trabalhamos para o nosso projeto, infelizmente ele nao esta funcionando 100% no dominio online, o banco de dados nao aceita utf8 por algum motivo, o endereco hospedado: www.nfldazoera.webutu.com.
Eu nao consegui fazer de jeito nenhum o banco de dados aceitar utf8 online hospedado.
Fizemos todos os itens que a senhora pediu, senhas criptografas com session, pagina de login, pagina de cadastro, área de usuário e pagina de administrador.

Giovanirose committed:
Merge pull request #4 from Giovanirose/entrega-final

Entrega Final - GRAND FINALE
